site/src/server.js

```diff
@@ -92,8 +90,11 @@ if (process.env.GITHUB_CLIENT_ID) {
 	});
 }
 
+app.get('/foo', (req, res) => {
```

Comment:
🙂

Reply:
Ughh hate not being on my real computer. All my tricks are gone 😢 

Force pushed. Thanks